packages/frontend/src/hooks/echarts/useEchartsCartesianConfig.ts

```diff
@@ -1433,13 +1426,26 @@ const useEchartsCartesianConfig = (
                 : getResultValueArray(rows, true);
         try {
             if (!itemsMap) return results;
-
             const xFieldId = validCartesianConfig?.layout?.xField;
             if (xFieldId === undefined) return results;
+            const { min, max } = axes.xAxis[0];
```

Comment:
Not sure if related to this line, but the range was not applied when I flipped the axis

<img width="1790" alt="Screenshot 2024-01-25 at 13 33 48" src="https://github.com/lightdash/lightdash/assets/9117144/fc91fca2-524d-4d15-9866-7bf126b9807d">

Reply:
ticket created https://github.com/lightdash/lightdash/issues/8262